distmv/statdist.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/gonum/stat"
 )
 
+type DistNormaler interface {
```

Comment:
8082 is a problem, I agree, but the general principle is to define interfaces where they will be used (or at least something that could be described as the closest common dependency lending from evolutionary terminology). At the moment your packages are the only dependents. I would say that if other packages show a need for these interfaces because of use like what is described in 8082 (rather than just general use that doesn't cause syntactic mismatch) then these could be added, but at the moment it seems premature.

Possibly naively, I would render your use here (from the definitions of the interface) as a closer or method value.